<launch>
  <node pkg="ros_imc_broker" type="ros_imc_adapter_node" name="ros_imc_adapter_node" output="screen">
    <param name="system_name" value="bogus"/>
    <param name="system_type" value="uuv"/>
    <param name="system_imc_id" value="8226"/> <!--0x2022-->

    <param name="enable_loopback" value="True"/>

    <param name="udp_port" value="6002"/>
    <!--param name="udp_port_tries" value="10"/-->

    <!--param name="multicast_addr" value="224.0.75.69"/-->
    <!--param name="multicast_port" value="30100""/-->
    <!--param name="multicast_port_range" value="5"/-->

    <param name="static_destinations_addrs" value="172.17.0.1:6001"/>
  </node>
  <node pkg="ros_imc_broker" type="params_changer_node" name="settings" output="screen"/>
</launch>
